Makefile

```diff
@@ -638,7 +638,7 @@ $(ROCKSDB_DIR)/Makefile: $(C_DEPS_DIR)/rocksdb-rebuild | bin/.submodules-initial
 	@# NOTE: If you change the CMake flags below, bump the version in
 	@# $(C_DEPS_DIR)/rocksdb-rebuild. See above for rationale.
 	cd $(ROCKSDB_DIR) && CFLAGS+=" $(sse)" && CXXFLAGS+=" $(sse)" && cmake $(xcmake-flags) $(ROCKSDB_SRC_DIR) \
-	  $(if $(findstring release,$(BUILDTYPE)),-DPORTABLE=ON) -DWITH_GFLAGS=OFF \
+	  -DPORTABLE=ON -DWITH_GFLAGS=OFF \
```

Comment:
Maybe add a comment explaining what this will do. I assume it will prevent use of march=native?

Reply:
Yes, this prevents use of `-march=native`. Comment added.